chore(release): update changelog for delayed v0.19.0 release (#1911)
waku-org/nwaku: CHANGELOG.md

## 2023-08-16 v0.19.0

> > Note that the `--topic` CLI option is being depreaced in favour of two more spefici options `--pubsub-topic` and `--content-topic`.

Reviewer: ```suggestion
> Note that the `--topic` CLI option is being deprecated in favor a more specific option `--pubsub-topic`.
```

I don't foresee that `content-topic` will replace `pubsub-topic` - it will be an alternative, sure, when autosharding is in place, but I don't think we have the context to mention this here.

Reviewer: Also, as much as it pains me, we try to use US English for publications, so "favor" instead of "favour" 😢

| [`13/WAKU2-STORE`](https://rfc.vac.dev/spec/13/) | `draft` | `/vac/waku/store/2.0.0-beta4` |
| [`19/WAKU2-LIGHTPUSH`](https://rfc.vac.dev/spec/19/) | `draft` | `/vac/waku/lightpush/2.0.0-beta1` |

The Waku v1 implementation has been removed from this repository and can be found in a separate [Waku Legacy](https://github.com/waku-org/waku-legacy) repository.

## Upgrade instructions

> * The `--topic` CLI option is being depreaced in favour of two more spefici options `--pubsub-topic` and `--content-topic`. The `--topic` option will be available for next 2 releases with a deprecation note.

Reviewer: See my notes for the same sentence at the top level.
